src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,30 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import React from 'react';
+import classNames from 'classnames';
+import './TodoInfo.scss';
+
+import { Todo } from '../../types/Todo';
+
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    title,
+    completed,
+    user,
+  } = todo;
+
+  return (
+    <article className={classNames('TodoInfo', {
+      'TodoInfo--completed': completed,
+    })}
+    >
+      <h2 className="TodoInfo__title">{title}</h2>
+
+      {user && <UserInfo user={user} key={todo.userId} />}
```

Comment:
you don't really need a key here, you're not mapping users
```suggestion
      {user && <UserInfo user={user}/>}
```